### PR TITLE
Prep for 6.6

### PR DESCRIPTION
**Pull request by yadij**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+	- Bug 5328: Fix ESI build with libxml2 v2.12.0
+	- Bug 5319: QOS Netfilter MARK preservation is always disabled
+	- Bug 5318: peer_digest.cc:399: "fetch->pd && receivedData.data"
+	- Bug 5317: FATAL attempt to read data from memory
+	- Bug 5154: Do not open IPv6 sockets when IPv6 is disabled
+	- FTP: Ignore credentials with a NUL-prefixed username
+	- log_db_daemon: Fix DSN construction
+	- Limit the number of allowed X-Forwarded-For hops
+	- Do not update StoreEntry expiration after errorAppendEntry()
+	- Just close after a write(2) response sending error
```
Comment (kinkie) [inline ChangeLog:12]:
```suggestion
	- more gracefully handle errors when sending data during respons
```

**Review by kinkie — APPROVED** [review]

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+	- Bug 5328: Fix ESI build with libxml2 v2.12.0
+	- Bug 5319: QOS Netfilter MARK preservation is always disabled
+	- Bug 5318: peer_digest.cc:399: "fetch->pd && receivedData.data"
+	- Bug 5317: FATAL attempt to read data from memory
+	- Bug 5154: Do not open IPv6 sockets when IPv6 is disabled
+	- FTP: Ignore credentials with a NUL-prefixed username
+	- log_db_daemon: Fix DSN construction
+	- Limit the number of allowed X-Forwarded-For hops
+	- Do not update StoreEntry expiration after errorAppendEntry()
+	- more gracefully handle errors when sending data during respons
```
Comment (rousskov) [inline ChangeLog:12]: Just to fix spelling/typo:

```suggestion
	- more gracefully handle errors when sending data during response
```

To improve wording _and_ avoid sounding like the _opposite_ of what commit b850f8b actually does:

```suggestion
	- improve handling of response sending errors
```

N.B. Commit b850f8b harshly terminates all associated transactions.

**Review by rousskov — APPROVED** [review]
Approving (to avoid delaying a v6 release) despite my long-standing dissatisfaction with how change log entries are worded and how ChangeLog changes are managed.

**Comment by rousskov** [comment]
> @rousskov , @kinkie - Anubis seems to be stuck. The CI pr-auto job has not triggered in the past 4 days.

I see no signs of Anubis being stuck. There are a few PRs that are cleared for merging and can be merged. Anubis is waiting for the oldest of those PRs to pass tests (for the commit staged on the auto branch).

* Right now, that oldest PR is #1604. That PR is waiting for Jenkins staged commit tests (with 13 out of 15 checks already completed).
* When you wrote the above comment, that oldest PR was probably #1603. AFAICT, that PR was waiting for Jenkins staged commit tests as well. Based on GitHub-reported checks for that staged commit (4299d1e), some required Jenkins tests did not start or were not reported to GitHub. Jenkins job [1773](https://build.squid-cache.org/job/5-pr-auto/1773/) has several [failed tests](https://build.squid-cache.org/job/5-pr-auto/1773/COMPILER=gcc,OS=debian-testing,label=docker-build-host/console) that are not reported to GitHub -- I have whined about such "unreported" or "invisible" or "silent" Jenkins failures many times before.

It will take a while for other older PRs to merge, especially if nobody babysits Jenkins. If you think this is an emergency, you can merge this unanimously approved PR manually now, without waiting for those other PRs.

**Comment by kinkie** [comment]
Oh, interesting, I wasn't aware Anubis was strictly in order when merging PRs.
I'll see why jenkins is stuck in that PR than, brecause it's not doing anything right now

**Pull request closed without merging**